perf/roles/trex/tasks/main.yml

```diff
@@ -9,29 +9,47 @@
       - python3
       - rdma-core-devel
       - zlib-devel
+      - driverctl
     state: installed
   tags: install
 
+- name: Install kernel-devel
+  yum:
+    name: kernel-devel-{{ ansible_kernel }}
+    state: present
+  tags: install
+
+- name: Install Development Tools group
+  yum:
+    name: "@Development Tools"
+    state: present
+  tags: install
```

Comment:
Why do you need this?

Reply:
I faced this issue on my DSAL systems. But on NFV machines I never faced this issue. 

Usually we have pre-compiled igb_uio.ko for common Kernels of supported OS.

Ref: 
If you have different Kernel (due to update of packages or slightly different OS version), you will need to compile that module.
Simply follow the instructions printed above.
Note: you might need additional Linux packages for that:

Fedora/CentOS:

sudo yum install kernel-devel-`uname -r`
sudo yum group install "Development tools"

Reply:
We don't need igb_uio.ko. We should use vfio-pci.ko for intel nics.

Reply:
Updated now .. got confused between vfio-pci and igb_uio